core-concepts/plugins.md

```diff
@@ -345,40 +346,40 @@ declare module 'pinia' {
 }
 ```
 
-### Typing new creation options
+### Tipar nuevas opciones de creación {#typing-new-creation-options}
 
-When creating new options for `defineStore()`, you should extend the `DefineStoreOptionsBase`. Differently from `PiniaCustomProperties`, it only exposes two generics: the State and the Store type, allowing you to limit what can be defined. For example, you can use the names of the actions:
+Cuando creas nuevas opciones para `defineStore()`, deberás extender el `DefineStoreOptionsBase`. A diferencia de `PiniaCustomProperties`, solo expone dos genéricos: el tipo Estado y el tipo Almacén, permitiéndote limit que puede ser definido. Por ejemplo, puedes usar los nombres de las acciones:
```

Comment:
"Limit" is not translated into Spanish. 
Perhaps it could be translated as follows:

Al crear nuevas opciones para `defineStore()`, debe extender la `DefineStoreOptionsBase`. A diferencia de `PiniaCustomProperties`, sólo expone dos genéricos: el Estado y el tipo de Almacén, lo que le permite limitar lo que se puede definir. Por ejemplo, puede utilizar los nombres de las acciones:

What do you think?

Reply:
yeah that's much better